docs/rest-api.md

```diff
@@ -0,0 +1,136 @@
+# REST API in Azure SignalR Service
+
+- [Typical Server-less Architecture](#serverless)
+- [API](#api)
+    - [Broadcast message to all clients](#broadcast)
+    - [Broadcast message to a group](#broadcast-group)
+    - [Send message to a user](#send-user)
+    - [Add a user to a group](#add-user-to-group)
+    - [Remove a user from a group](#remove-user-from-group)
+- [Using REST API](#using-rest-api)
+    - [Authentication](#authentication)
+        - [Signing Algorithm and Signature](#signing)
+        - [Claims](#claims)
+    - [User-related REST API](#user-api)
+    - [Sample](#sample)
+
+On top of classical client-server pattern, Azure SignalR Service provides a set of REST APIs, so that you can easily integrate real-time functionality into your server-less architecture.
+
+<a name="serverless"></a>
+## Typical Server-less Architecture with Azure Functions

```

Comment:
Add a note that 1.1.0-preview client cannot work in serverless scenario

Reply:
Sure.